src/components/signup/SignupState.js

```diff
@@ -129,6 +129,13 @@ const Signup = ({ navigation, screenProps }: { navigation: any, screenProps: any
         break
 
       case 'goToPhone':
+        API.checkWeb3Email({
+          email: w3User.email,
+          token: web3Token,
+        }).catch(e => {
+          showErrorDialog('Email verification failed', e)
```

Comment:
always log.error errors
use this format: log.error(<error message>,e.message,e,other params...)

Reply:
ok